pages/index.tsx

```diff
@@ -1,15 +1,18 @@
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
```

Comment:
Can we decide on using ArrowForwardIcon or ArrowForwardIos, and just use one throughout?

Reply:
Although the naming is similar, they are two different icons.